### DISCUSSION TITLE
Debug mode with logfiles

### DISCUSSION THREAD

**Discussion by DevScherer** [jomjol/AI-on-the-edge-device#1237] [General] [answered]
@caco3 
Is it possible to retain more than 30 days of logfiles? 
I will build my gas-metre at the weekend and came across this question now... 
And thanks to all for this great tool! Looks really cool. 

Is there a tutorial on how I could contribute --> language, tools needed? I am thinking about a Telegram-Bot integration without any 3rd-party tools / gateways but everything from the client. 

Cheers
DevScherer

**Comment by caco3** [comment, marked as answer]
Where do you see a limit of 30 days' I configured mine to 90 and I don't see any restriction (but I don't know what happens when the SD-Card gets full.

**Comment by caco3** [comment]
As for the contributions, there is not really a tutorial. Maybe check the wiki. For how to build it, there also is a readme: https://github.com/jomjol/AI-on-the-edge-device/blob/master/code/README.md 

Just fork it and create a pull request if you have something and we can have a look on it.
